src/components/VueMoney.js

```diff
@@ -14,23 +14,23 @@ export default {
         };
     },
     methods: {
-        onInput(e) {
-            const value = e.target.value.replaceAll(",", "");
+        onInput(event) {
+            const value = event.target.value.replaceAll(",", "");
             const formatMoney = moneyFormat(value);
 
             if (
                 !checkToOnlyNumber(value) ||
                 (this.minMax.length && (this.minMax[0] > +value || +value > this.minMax[1]))
             ) {
-                e.target.value = this.oldValue;
+                event.target.value = this.oldValue;
             } else {
-                e.target.value = formatMoney;
+                event.target.value = formatMoney;
                 this.oldValue = formatMoney;
             }
         },
     },
-    render(h) {
-        return h("input", {
+    render(createElement) {
+        return createElement("input", {
             attrs: { type: "text", name: "money-input", class: "vf-money" },
             on: { input: this.onInput },
         });
```

Comment:
I used "h" because I think it will be easier. In terms of understanding. Using createElement is not required by the standard.